integration-tests/common-test-code/src/main/java/com/vaadin/flow/quarkus/it/TestServlet.java

```diff
@@ -6,9 +6,8 @@
 import com.vaadin.quarkus.QuarkusVaadinServlet;
 
 @WebServlet(urlPatterns = "/*", name = "TestServlet", asyncSupported = true, initParams = {
-        @WebInitParam(name = "org.atmosphere.websocket.suppressJSR356", value = "true"),
-        // TODO: Enable dev mode gizmo when websockets are in use
-        @WebInitParam(name = "devmode.gizmo.enabled", value = "false")})
+        @WebInitParam(name = "vaadin.productionMode", value = "false"),
+        @WebInitParam(name = "devmode.gizmo.enabled", value = "true") })
```

Comment:
This is basically the same as default so a `true` should not be needed.

Also this conflicts with the production mode tests that do not support the gizmo.

Reply:
As mentioned above, maybe we can completely drop the servlet .